website/src/pages/user/createGroup.astro

```diff
@@ -7,8 +7,9 @@ import { getAccessToken } from '../../utils/getAccessToken';
 const accessToken = getAccessToken(Astro.locals.session)!;
 
 const clientConfig = getRuntimeConfig().public;
+const searchParams = Astro.url.searchParams.toString();
```

Comment:
(though this could be done better IMO it's not worth another round of iteration)

Reply:
(I did confirm that the functionality still works)